src/client/pages/ApplicationError/ApplicationError.tsx

```diff
@@ -0,0 +1,224 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { FallbackProps } from 'react-error-boundary';
+import classNames from 'classnames';
+import styles from './ApplicationError.module.scss';
+import footerStyles from '../../components/MainFooter/MainFooter.module.scss';
+import { AppRoutes, ExternalUrls } from '../../../universal/config';
+import { useDesktopScreen } from '../../hooks';
+import Heading from '../../components/Heading/Heading';
+import { PageContent, TextPage } from '../../components/Page/Page';
+import PageHeading from '../../components/PageHeading/PageHeading';
+import { ReactComponent as AmsterdamLogoLarge } from '../../assets/images/logo-amsterdam-large.svg';
+import { ReactComponent as AmsterdamLogo } from '../../assets/images/logo-amsterdam.svg';
+import { Linkd, LinkdInline } from '../../components';
+import { isExternalUrl } from '../../../universal/helpers';
+import { useUsabilla } from '../../hooks/useUsabilla';
+
+const LANDSCAPE_SCREEN_RATIO = 0.25;
+const PORTRAIT_SCREEN_RATIO = 0.4;
+
+const BOTTOM_LINKS = [
+  {
+    title: 'Over deze site',
+    to: 'https://www.amsterdam.nl/overdezesite/',
+  },
+  {
+    title: 'Privacy',
+    to: 'https://www.amsterdam.nl/privacy/',
+  },
+  {
+    title: 'Webarchief',
+    to: 'https://amsterdam.archiefweb.eu/',
+  },
+];
```

Comment:
Aangezien de content van de footer uit het CMS komt denk ik dat het een matig idee is om die content hier te dupliceren. Ik snap dat de contactmogelijkheden er wel staan maar deze links lijken mij overbodig voor deze pagina. Kun je dat met Bram bespreken?